.rubocop.yml

```diff
@@ -2,6 +2,8 @@ Documentation:
    Enabled: false
 LineLength:
    Max: 99
+BlockLength:
```

Comment:
![a483ca0b413153496e7ac49ca013334d380dbafeb146d5715252c96f4e683680](https://user-images.githubusercontent.com/4212982/29222959-1d4450fa-7e9b-11e7-8c5e-b4c10f29c93e.jpg)

As an alternative you can do something like:

```
Metrics/BlockLength:
  Exclude:
    - "**/*_spec.rb"

```